eta/core/serial.py

```diff
@@ -824,6 +835,36 @@ def from_dict(cls, d, **kwargs):
         elements = [cls._ELE_CLS.from_dict(dd) for dd in d[cls._ELE_ATTR]]
         return cls(**etau.join_dicts({cls._ELE_ATTR: elements}, kwargs))
 
+    @classmethod
+    def from_element_patt(cls, pattern, pattern_type=NUMERIC):
```

Comment:
I suggest adding arguments `*args, **kwargs` that will be passed to the `from_json()` method.  See [here](https://github.com/voxel51/eta/blob/develop/eta/core/serial.py#L388).

Reply:
happy now?!